BrowserKit/Sources/ComponentLibrary/Buttons/PaddedSwitch.swift

```diff
@@ -1,52 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/
-
-import Foundation
-import Common
-import UIKit
-
-public class PaddedSwitch: UIView, ThemeApplicable {
```

Comment:
I wouldn't remove that class, since it's part of the component library and it's part of the [mobile design components](https://www.figma.com/design/YIbYarab5aYte7KbK0bcVw/iOS-Components?node-id=1202-53177&p=f&t=Snm03pYfNEvLK9mb-0)